amoro-format-paimon/src/test/java/org/apache/amoro/formats/paimon/PaimonHadoopCatalogTestHelper.java

```diff
@@ -45,6 +45,7 @@ public class PaimonHadoopCatalogTestHelper extends AbstractFormatCatalogTestHelp
           .primaryKey("id", "age")
           .partitionKeys("age")
           .option("amoro.test.key", "amoro.test.value")
+          .option("alter-column-null-to-not-null.disabled", "false")
```

Comment:
Paimon version 1.2.0 has restrictions on changing non-null types to null types; this configuration ```alter-column-null-to-not-null.disabled``` needs to be ```false```, otherwise this test ```TestPaimonServerTableDescriptor##tableOperationsChangeColumnRequired``` cannot be passed.